src/main/java/mops/feedback/formulare/AntwortenDAO.java

```diff
@@ -0,0 +1,25 @@
+package mops.feedback.formulare;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.Getter;
+import lombok.Setter;
+import org.springframework.data.annotation.Id;
+import org.springframework.data.relational.core.mapping.Table;
+
+@Data
+@Table
+@AllArgsConstructor
+@Setter
+@Getter
```

Comment:
@Table needs the name of the Table
@Data implies @Setter, @Getter and @AllArgsConstructor